src/hooks/useUserGauge.ts

```diff
@@ -54,6 +57,21 @@ export default function useUserGauge(gaugeAddress?: string): UserGauge | null {
     userGaugeRewards?.claimableExternalRewards.length,
   )
 
+  const boost =
+    gauge.gaugeBalance != null &&
+    !gauge.gaugeBalance.isZero() &&
+    gauge.workingBalances != null &&
+    !gauge.workingBalances.isZero()
```

Comment:
```
gauge?.gaugeBalance?.gt(Zero) &&
gauge?.workingBalances?.gt(Zero)
```
this should accomplish similar if you wanted to simplify

Reply:
might just be 
```
gauge.gaugeBalance?.gt(Zero) &&
gauge.workingBalances?.gt(Zero)
```

Reply:
Good call. Will use it now.